bin/citrea/src/test_rpc.rs

```diff
@@ -5,7 +5,7 @@ use reqwest::header::CONTENT_TYPE;
 use sha2::Digest;
 use sov_db::ledger_db::{LedgerDB, SharedLedgerOps};
 use sov_db::rocks_db_config::RocksdbConfig;
-use sov_db::schema::types::StoredTransaction;
+use sov_db::schema::types::soft_confirmation::StoredTransaction;
```

Comment:
I would suggest this PR to land after https://github.com/chainwayxyz/citrea/pull/1825